### PR TITLE
Django - good practices and some work

### PR DESCRIPTION
**Pull request by rodgomesc**

### CODE REVIEW

File: core/settings.py [hunk 1]
```diff
@@ -11,9 +11,8 @@
 SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=False, cast=bool)
+DEBUG = config('DEBUG', default=True, cast=bool)
```
Comment (rphbc) [inline core/settings.py]: @rodgomesc I don't think this is a good idea. On a deploy if there is no enviroment variables the application will go to production with DEBUG mode ON.

File: core/settings.py [hunk 2]
```diff
@@ -11,9 +11,8 @@
...
 
-ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
```
Comment (rphbc) [inline core/settings.py]: @rodgomesc this is a security issue. Why take out this parameter?
ref: https://docs.djangoproject.com/en/2.1/ref/settings/#allowed-hosts

File: core/settings.py (hunk 1 above)
Comment (rodgomesc) [reply core/settings.py]: sorry, I changed this for testing and forgot to update

File: core/settings.py (hunk 2 above)
Comment (rodgomesc) [reply core/settings.py]: sorry, I changed this for testing and forgot to update

File: apps/api/views.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        lat, lng = serializer.data['lat'], serializer.data['lng']
+        vector_position = Position(lat, lng).calc_vector()
+        return Response(vector_position, status=status.HTTP_200_OK)
+
+
+calculatecoordinate = CalculateCoordinate.as_view()
+
+
+
+def get_raw_data():
```
Comment (cacarrara) [inline apps/api/views.py]: O `crawler` não é uma view. Não deveria estar aqui.

File: apps/api/urls.py [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
+from apps.api.views import calculatecoordinate, crawler
 
 app_name = 'api'
 
 urlpatterns = [
-    path('calculate', calculatecoordinate, name='coordinate_calculate')
-]
\ No newline at end of file
+    path('calculate', calculatecoordinate, name='coordinate_calculate'),
+    path('crawler/', crawler),
```
Comment (cacarrara) [inline apps/api/urls.py]: Crawler não é acessado através de um endpoint. Não precisa dessa definição de url.

File: static/data/persons.csv [hunk 5]
```diff
@@ -0,0 +1,292 @@
+Adail Dos Santos Junior,28/01/2019 - 10:30
```
Comment (cacarrara) [inline static/data/persons.csv]: Esse csv não precisa ser versionado.

File: apps/api/urls.py [hunk 6]
```diff
@@ -1,9 +1,9 @@
 from django.urls import path
 
-from api.views import calculatecoordinate
+from apps.api.views import calculatecoordinate, crawler
```
Comment (thiagolopes) [inline apps/api/urls.py]: you don't need `crawler` import anymore.

File: apps/api/views.py [hunk 7]
```diff
@@ -31,3 +32,4 @@ def get_elevation(lat, lng):
 
 
 calculatecoordinate = CalculateCoordinate.as_view()
+
```
Comment (thiagolopes) [inline apps/api/views.py]: you can remove this blank line :lipstick:

File: apps/api/urls.py (hunk 6 above)
Comment (rodgomesc) [reply apps/api/urls.py]: Opps

File: apps/api/views.py (hunk 7 above)
Comment (rodgomesc) [reply apps/api/views.py]: this is a pep8 recomendation

File: apps/api/urls.py (hunk 6 above)
Comment (rodgomesc) [reply apps/api/urls.py]: fixed on https://github.com/sosbrumadinho/brumadinho_location/pull/68/commits/eaab178b07732a46f48f882691bb20ead7c3c3c6

File: apps/api/views.py (hunk 7 above)
Comment (thiagolopes) [reply apps/api/views.py]: some files in this projects don't follow `W292`, but no problem, this is aesthetic

**Pull request merged**